app/src/main/java/com/kkolontay/theworld/model/Country.kt

```diff
@@ -1,19 +1,22 @@
 package com.kkolontay.theworld.model
 
+import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
 data class CountryName(
-     val common: String)
+    @Json(name = "common") val common: String)
```

Comment:
Minor but I don't think you need the `@Json` annotations on any of these models here in this file.  The names all match the variables.

Reply:
Also, for each of these data classes the ending parentheses can go on a new line.

```kotlin
JsonClass(generateAdapter = true)
data class CountryName(
    val common: String
)
```

It's easier to read where the class ends this way.